test/built-ins/RegExp/prototype/source/this-val-regexp-prototype.js

```diff
@@ -3,15 +3,12 @@
 /*---
 esid: sec-get-regexp.prototype.source
 description: >
-  Return value of `undefined` when the "this" value is the RegExp prototype
-  object
+  Return "(?:)" when the `this` value is the RegExp.prototype object
 info: |
   1. Let R be the this value.
   2. If Type(R) is not Object, throw a TypeError exception.
   3. If R does not have an [[OriginalFlags]] internal slot, then
      a. If SameValue(R, %RegExpPrototype%) is true, return "(?:)".
 ---*/
 
-var get = Object.getOwnPropertyDescriptor(RegExp.prototype, 'source').get;
-
-assert.sameValue(get.call(RegExp.prototype), '(?:)');
+assert.sameValue(RegExp.prototype.source, '(?:)');
```

Comment:
while I really like the new description, the current assertion change might provide a false positive if the descriptor returns a data descriptor, using `value` instead of a get call. I assume the current original test is doing an explicit get call for this purpose.